code/modules/mob/living/simple_animal/hostile/gym_rats.dm

```diff
@@ -429,52 +480,48 @@
 				wander = TRUE
 				speed = 2
 
-/mob/living/simple_animal/hostile/retaliate/gym_rat/roid_rat/Calm()
-	enemies.Cut()
-	LoseTarget()
-	src.say(pick("YOU AIN'T NUTHING.","BETTER RUN, TINY MAN."))
-	src.visible_message("<span class='notice'>[src] squeaks softly and calms down.</span>")
-
-/mob/living/simple_animal/hostile/retaliate/gym_rat/roid_rat/Retaliate()
-	if(!stat)
-		..()
-		src.say(pick("TIME TO STEP INTO THE SQUARE CIRCLE, SON.","YOU CALLED DOWN THE THUNDER.","GET READY FOR THE BIG GUNS.","YOU EYEBALLING ME?"))
-
-/mob/living/simple_animal/hostile/retaliate/gym_rat/roid_rat/attackby(var/obj/item/O as obj, var/mob/user as mob) // Feed the roid rat some food
+/mob/living/simple_animal/hostile/retaliate/gym_rat/roid_rat/attackby(var/obj/item/weapon/reagent_containers/food/snacks/F as obj, var/mob/user as mob) // Feed the roid rat some food
 	if(stat == CONSCIOUS)
-		if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/cheesewedge)) // Cheesewedges heal it a bit
-			Calm()
+		if(istype(F, /obj/item/weapon/reagent_containers/food/snacks/cheesewedge_scraps)) // Cheesewedge scraps heal it a bit
 			health+=10
 			playsound(src, 'sound/items/eatfood.ogg', rand(10,50), 1)
-			visible_message("<span class='notice'>[user] feeds \the [O] to [src]. It squeaks loudly.</span>")
+			visible_message("<span class='notice'>[user] feeds \the [F] to [src]. It squeaks loudly.</span>")
 			var/image/heart = image('icons/mob/animal.dmi',src,"heart-ani2")
 			heart.plane = ABOVE_HUMAN_PLANE
 			flick_overlay(heart, list(user.client), 20)
 			flick(icon_eat, src)
-			qdel(O)
-		else if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/sliceable/cheesewheel))
-			Calm()
-			health+=50
+			qdel(F)
+		if(istype(F, /obj/item/weapon/reagent_containers/food/snacks/cheesewedge)) // Cheesewedges heal it a bit
+			health+=15
 			playsound(src, 'sound/items/eatfood.ogg', rand(10,50), 1)
-			visible_message("<span class='notice'>[user] feeds \the [O] to [src]. It squeaks loudly.</span>")
+			visible_message("<span class='notice'>[user] feeds \the [F] to [src]. It squeaks loudly.</span>")
 			var/image/heart = image('icons/mob/animal.dmi',src,"heart-ani2")
 			heart.plane = ABOVE_HUMAN_PLANE
 			flick_overlay(heart, list(user.client), 20)
 			flick(icon_eat, src)
-			qdel(O)
-		else if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/meat)) // Meat heals less, but packs on some extra maximum hp
-			Calm()
+			qdel(F)
+		else if(istype(F, /obj/item/weapon/reagent_containers/food/snacks/sliceable/cheesewheel))
+			health+=75
```

Comment:
instead of pasting this, make feeding it into a proc that you can call from both places

Reply:
Okay, should be addressed. The pr now removes more lines than it adds.

All the lines under attackby are now replaced with calling of the "chowdown" proc.